package.json

```diff
@@ -1,10 +1,11 @@
 {
-  "name": "vanilla-contracts",
-  "version": "1.1.0",
+  "name": "@vanilladefi/trade-contracts",
+  "version": "1.1.5",
   "description": "Smart contracts for vanilladefi.com",
+  "main": "./typechain/vanilla_v1.1/index.ts",
+  "type": "module",
   "scripts": {
-    "preinstall": "npx only-allow pnpm",
-    "postinstall": "pnpm run generate:typechain",
+    "postinstall": "npm run compile:sol & npm run generate:typechain",
```

Comment:
Shouldn't this be `prepare` or something instead of `postinstall`. `postinstall` also runs when some other packages (`sdk`) installs it via `npm install ...`

Reply:
I think postinstall is actually the best way to generate the typechain definitions, if it works without requiring a pnpm install